src/internal_events/http.rs

```diff
@@ -68,39 +68,70 @@ impl InternalEvent for HttpEventsReceived<'_> {
 
 #[derive(Debug)]
 pub struct HttpBadRequest<'a> {
-    pub error_code: u16,
-    pub error_message: &'a str,
+    pub code: u16,
+    pub message: &'a str,
+}
+
+impl<'a> HttpBadRequest<'a> {
+    fn error_code(&self) -> String {
+        http_error_code(self.code)
+    }
 }
 
 impl<'a> InternalEvent for HttpBadRequest<'a> {
     fn emit_logs(&self) {
         warn!(
             message = "Received bad request.",
-            code = ?self.error_code,
-            error_message = ?self.error_message,
+            error = %self.message,
+            error_code = %self.error_code(),
```

Comment:
Should this get a translated HTTP error code too?

Reply:
🤔 I don't think I get what you'd like me to do

Reply:
I missed the parenthesis on the `error_code` and thought you were logging the numeric code instead of the string. This is fine.